theme/base-2016/partials/_footer.twig

```diff
@@ -17,6 +17,8 @@
 
 </footer>
 
-<script src="{{ asset('js/jquery.min.js', 'theme') }}"></script>
+{# Extensions and copypasta gleaned from other sources tend to cause multiple jQuery
+   includes. We only include it here conditionally, if not done so already. #}
+<script>window.jQuery || document.write("<script src='{{ asset('js/jquery.min.js', 'theme') }}'><\/script>");</script>
```

Comment:
Interesting read, but i am not convinced it will block: 

> The script is not hosted on the same site. In other words, Chrome will not intervene for scripts with a matching eTLD+1 (e.g. a script hosted on js.example.org inserted on www.example.org).